applications/simulate_prod.py

```diff
@@ -0,0 +1,332 @@
+    on shower and array simulations.
+
+    Attributes
+    ----------
+    config_file: str
+        Name of simulation configuration file.
+    primary_config: str
+        Name of the primary selected from the configuration file.
+
+    Returns
+    -------
+    str
+        Label of simulation configuration.
+    dict
+        Configuration of shower simulations.
+    dict
+        Configuration of array simulations.
+
+    """
+
+    try:
+        with open(config_file) as file:
+            config_data = yaml.load(file)
+    except FileNotFoundError:
+        logger.error(f"Error loading simulation configuration file from {config_file}")
+        raise
+
+    label = config_data.pop("label", "test_run")
+    default_data = config_data.pop("default", {})
+
+    for primary, primary_data in config_data.items():
+
+        if primary_config is not None and primary != primary_config:
+            continue
+
+        this_default = copy(default_data)
+
+        config_showers = copy(this_default.pop("showers", {}))
+        config_arrays = copy(this_default.pop("array", {}))
+
+        # Grabbing common entries for showers and array
+        for key, value in primary_data.items():
+            if key in ["showers", "array"]:
+                continue
+            config_showers[key] = value
+            config_arrays[key] = value
+
+        # Grabbing showers entries
+        for key, value in primary_data.get("showers", {}).items():
+            config_showers[key] = value
+        config_showers["primary"] = primary
+
+        # Grabbing array entries
+        for key, value in primary_data.get("array", {}).items():
+            config_arrays[key] = value
+        config_arrays["primary"] = primary
+
+        # Filling in the remaining default keys
+        for key, value in this_default.items():
+            config_showers[key] = value
+            config_arrays[key] = value
+
+    config_arrays["data_directory"] = config_showers["data_directory"]
+    config_arrays["site"] = config_showers["site"]
+    config_arrays["layout_name"] = config_showers["layout_name"]
+
+    return label, config_showers, config_arrays
+
+
+def _translate_from_azimuth_direction_to_phi(logger, from_azimuth_direction):
+    """
+    Translate the direction particles are coming from to an azimuth angle.
+
+    Attributes
+    ----------
+    from_azimuth_direction: str (north, south, east, west)
+        The direction particles are coming from.
+
+    Returns
+    -------
+    float (Astropy.Quantity)
+        The phi angle for CORSIKA configuration.
+
+    """
+
+    if from_azimuth_direction == "north":
+        return 0 * u.deg
+    if from_azimuth_direction == "south":
+        return 180 * u.deg
+    if from_azimuth_direction == "east":
+        return 90 * u.deg
+    if from_azimuth_direction == "west":
+        return 270 * u.deg
+
+    logger.error(f"The direction {from_azimuth_direction} to simulate from was not recognised")
```

Comment:
recognised x recognized? Do we have a preference for BE or AE?

Reply:
Generally we write code in AE but comments in BE. However, for `z` vs. `s` it is harder cause one can use BE while still using `z`. I usually use `s`, but not always and there's no way I will go through the code to make this consistent =)

Reply:
Uh, that is odd, why comments in BE and code in AE? I have asked the question before to propose a discussion but I did not want to propose you to make the entire code consistent. Let's then simply ignore this for the moment

Reply:
It's actually standard that code is in AE (see for example `color` in matplotlib). Not sure why, maybe it's cause usually AE is shorter or cause they contribute more code to the world. Either way, comments/docstrings are then a free choice and as European scientists we usually use BE everywhere.